ChoseYourOwnAdventure/Page.java

```diff
@@ -0,0 +1,40 @@
+package ChoseYourOwnAdventure;
+
+import java.util.ArrayList;
+
+public class Page {
+    private String text;
+    private Page p1;
+    private Page p2;
+
+
+    public Page(String text, Page p1, Page p2){
+        this.text = text;
+        this.p1 = p1;
+        this.p2 = p2;
+    }
+
+    public String getText() {
+        return text;
+    }
+
+    public Page nextpage(int choice){
+        switch (choice){

```

Comment:
try to avoid encoding things and choices into arbitrary values 
and if you do so - document it or try doing it with String - or even better - with Enums

Reply:
EXTRA TASK - learn Enum and use it in the snake program instead of the 1,2,3,4 directions